regression/cbmc/r_w_ok_inconsistent_valid/main.c

```diff
@@ -0,0 +1,62 @@
+#include <assert.h>
+#include <stdlib.h>
+
+char c1;
+
+char *p1;
+char *p2;
+char *p3;
+
+void test_global_pointer()
+{
+  char c2;
+
+  p1 = &c1;
+  p2 = &c2;
+  p3 = malloc(1);
+
+  __CPROVER_assume(
+    __CPROVER_r_ok(p1, 2) ||
+    __CPROVER_r_ok(p1 - 1, 1) ||
+    __CPROVER_r_ok(p1 + 1, 1) ||
+    __CPROVER_r_ok(p2, 2) ||
+    __CPROVER_r_ok(p2 - 1, 1) ||
+    __CPROVER_r_ok(p2 + 1, 1) ||
+    __CPROVER_r_ok(p3, 2) ||
+    __CPROVER_r_ok(p3 - 1, 1) ||
+    __CPROVER_r_ok(p3 + 1, 1));
```

Comment:
An english summary of what is being assumed here would be helpful (like a high level intention)

Reply:
I think it's just checking a bunch of invalid assumes and seeing that none of them are true?

Reply:
So add comment:

```
// Validate pointers with off by one validation assumptions
// (just before, just after the valid memory)
 // one byte to much
```

And perhaps, if it still works, group by pointer

Reply:
Added